backend/src/Edna/Web/Handlers.hs

```diff
@@ -1,22 +1,21 @@
+{-# LANGUAGE StarIsType #-}
```

Comment:
This addition is somewhat weird because you didn't add any new code to this module, you only removed some :suspect: